packages/web-twig/src/Resources/components/Breadcrumbs/stories/BreadcrumbsBasic.twig

```diff
@@ -0,0 +1,26 @@
+{% extends 'layout/default.html.twig' %}
+
+{% block content %}
+
+{% set items = [
+  {
+    title: 'Root',
+    url: '#rootUrl',
+  },
+  {
+    title: 'Category',
+    url: '#categoryUrl',
+  },
+  {
+    title: 'Subcategory',
+    url: '#subcategoryUrl',
+  },
+  {
+    title: 'Current page',
+    url: '#currentUrl',
+  },
+] %}
+
+<Breadcrumbs items={{items}} />
```

Comment:
```suggestion
<Breadcrumbs items={{ items }} />
```
I think we add spaces usually.

Reply:
it is also in the README above your changes. Could you format it there as well, please?

Reply:
Fixed.